docs/Reference/architecture/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING
+
+## Installation
+
+Cosmos Cash diagrams are based on PlantUML. 
+
+PlantUML can be installed on your local system through your platforms' package manager:
+
+MacOS, through [Homebrew](https://brew.sh/)
+
+```
+> brew install plantuml
+```
+
+Windows, through [Chocolatey](https://chocolatey.org/)
+
+```
+> choco install plantuml
+```
+
+Linux-based systems, through your distributions package manager
+
+## Contributing diagrams
+
+The source code for all diagrams SHOULD be stored in `./src` folder AND have the `.puml` suffix. Please refer to the [PlanUML documentation](https://plantuml.com) for further details.
```

Comment:
```suggestion
The source code for all diagrams SHOULD be stored in `./src` folder AND have the `.puml` suffix. See the [PlanUML documentation](https://plantuml.com) for further details.
```

Reply:
we don't want to use Please, we already are friendly and helpful, per https://developers.google.com/style/tone#politeness-and-use-of-please